website/www/site/data/en/join_beam.yaml

```diff
@@ -11,15 +11,20 @@
 # limitations under the License.
 
 - list:
-    - Join Slack
-    - Follow us on Twitter to get the latest information
-    - Follow us on YouTube to know first about the latest tutorials
-    - Follow meetup closest to your city
+    - { title: Mailing Lists, description: Join the official Apache Beam user and developer mailing lists. Learn how <a href="https://beam.apache.org/community/contact-us/">here</a> }
+    - { title: Slack Channel, description: "For real-time communication, join us on the #beam channels in the ASF workspace." }
+    - { title: Twitter account, description: Follow @ApacheBeam on Twitter for the most recent news on the project and community! }
+    - { title: YouTube channel, description: Follow our YouTube channel for the latest training and use cases. }
+    - { title: StackOverflow, description: Ask your Beam questions in StackOverflow using the apache-beam tag. }
+    - { title: Events, description: Follow our calendar in the homepage which has the latest events organized by the community. }
 
 - list:
-    - Policies?
-    - Documentation?
-
+    - { title: Read our Contribution Guide and Policies, description: Get familiar with our <a href="https://beam.apache.org/contribute/">contribution guide</a> and <a href="https://beam.apache.org/contribute/precommit-policies/">policies</a>. }
+    - { title: Create a Jira account, description: Create a <a href="https://issues.apache.org/jira/secure/Signup!default.jspa" target="_blank">Jira account</a> and send an email to dev@beam.apache.org to request comitter permissions. }
+    - { title: Find an area to contribute, description: Beam recognizes code and non-code contributions. Check out this Jira board for some inspiration. }
+    - { title: Contribute, description: To contribute code, you need to sign an Individual Contributor License Agreement (ICLA). Learn more here. }
```

Comment:
This is missing everything after "To contribute code", maybe it needs to be wrapped in quotes?

![image](https://user-images.githubusercontent.com/675055/106640349-ca365900-653a-11eb-9653-f5c678787450.png)

There also should be a link to some page about ICLAs in "Learn more *here*"

Reply:
Yes ur right, fixed